### PR TITLE
Added peopleSubscription API method

### PR DESCRIPTION
**Pull request by rankarpan**
- Resolves subscription status for people (subscribed or unsubscribed).
- Updates current subscription status for people (subscribed or unsubscribed).

### CODE REVIEW

File: resources/WebsitePeople.php [hunk 1]
```diff
@@ -124,6 +124,22 @@ public function updatePeopleData($websiteId, $peopleId, $data) {
...
+    return $result->decode_response()["data"];
+  }
+
+  public function updatePeopleSubscriptionStatus($websiteId, $peopleId, $data) {
+    $result = $this->crisp->_rest->patch(
+      "website/$websiteId/people/subscription/$peopleId",
+      json_encode($data)
+    );
+
+    return $result->decode_response();
```
Comment (eliottvincent) [inline resources/WebsitePeople.php]: Should be updated to `formatResponse` (see our last merge).

File: resources/WebsitePeople.php [hunk 2]
```diff
@@ -124,6 +124,22 @@ public function updatePeopleData($websiteId, $peopleId, $data) {
 
     return $result->decode_response();
   }
+
+  public function getPeopleSubscriptionStatus($websiteId, $peopleId) {
+    $result = $this->crisp->_rest->get(
+      "website/$websiteId/people/subscription/$peopleId"
+    );
+    return $result->decode_response()["data"];
```
Comment (eliottvincent) [inline resources/WebsitePeople.php]: Should be updated to `formatResponse` (see our last merge).

**Review by eliottvincent — CHANGES_REQUESTED** [review]
Hey thanks for the PR!

There are two small changes to do, I'll be able to merge after 👍

File: resources/WebsitePeople.php (hunk 1 above)
Comment (rankarpan) [reply resources/WebsitePeople.php]: @eliottvincent I have updated to `formatResponse`.

File: resources/WebsitePeople.php (hunk 2 above)
Comment (rankarpan) [reply resources/WebsitePeople.php]: @eliottvincent I have updated to `formatResponse`.

**Pull request merged**